docs-source/migrating-to-v5.md

```diff
@@ -0,0 +1,25 @@
+# Migration Guide
+
+## Migrating from v5 to v6
```

Comment:
Should this be "from v4 to v5" instead, based on the file name? Or different file name?

Reply:
Yes, fixed.